lib/LaTeXML/resources/XSLT/LaTeXML-picture-xhtml.xsl

```diff
@@ -146,7 +146,15 @@
         <!-- If foreignObject in a DIFFERENT namespace, copy as foreign markup -->
         <xsl:when test="local-name()='foreignObject'
                         and not(namespace-uri(child::*) = $SVG_NAMESPACE)">
```

Comment:
I should flag that this test is still wrong.

Reply:
Removing it is a nice experiment (since we have quite a number of tests).

Reply:
As far as I know, the foreignObject is only created via autoopen/autoclose, and the handler is not *supposed* to generate a foreignObject if the children are all SVG tags. On the other hand, I don't think the schema can enforce this, so there won't be any warnings if LaTeXML creates a redundant foreignObject by mistake.